packages/augur-core/source/contracts/reporting/OICash.sol

```diff
@@ -21,16 +21,16 @@ contract OICash is VariableSupplyToken, Initializable, IOICash {
 
     uint256 private constant MAX_APPROVAL_AMOUNT = 2 ** 256 - 1;
 
-    function initialize(IAugur _augur, IUniverse _universe, address _erc1820RegistryAddress) external beforeInitialized {
+    function initialize(IAugur _augur, IUniverse _universe) external beforeInitialized {
         endInitialization();
         augur = _augur;
         cash = ICash(_augur.lookup("Cash"));
         cash.approve(address(_augur), MAX_APPROVAL_AMOUNT);
         shareToken = IShareToken(_augur.lookup("ShareToken"));
         require(shareToken != IShareToken(0));
         universe = _universe;
-        erc1820Registry = IERC1820Registry(_erc1820RegistryAddress);
-        initialize1820InterfaceImplementations();
+
+        initializeMKRShutdownHandler(_augur.lookup("DaiVat"), _augur.lookup("Cash"));
```

Comment:
Isn't this line related to an unmerged branch?  Perhaps a mistake when fiddling with branch changes?

Reply:
Note: Same question for several other files.